src/nupic/algorithms/TemporalMemory.hpp

```diff
@@ -198,8 +215,19 @@ namespace temporal_memory {
    * @param learn
    * If true, segment activations will be recorded. This information is
    * used during segment cleanup.
+   *
+   * @param extraActive
+   * Vector of active external predictive inputs.  External inputs must be cell
+   * indexes in the range [0, extra).
+   *
+   * @param extraWinners
+   * Vector of winning external predictive inputs.  When learning, only these
+   * inputs are considered active.  External inputs must be cell indexes in the
+   * range [0, extra).
```

Comment:
isn't that extWinners is a subset of extActive?

Reply:
I will add this to the documentation.